css/style.css

```diff
@@ -89,11 +89,6 @@ h1 {
   text-align: right;
 }
 
-table {
-  text-align: center;
-  margin: 1em 0;
-}
-
 thead .sorting {
```

Comment:
We are not using any table in the HTML. So all table references in the CSS have to disappear.